packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -383,13 +383,15 @@ class FlutterDriver {
   final int _driverId;
 
   /// Client connected to the Dart VM running the Flutter application
-  final VMServiceClient _serviceClient;
+  final VMServiceClient serviceClient;
 
   /// JSON-RPC client useful for sending raw JSON requests.
   final rpc.Peer _peer;
 
-  /// The main isolate hosting the Flutter application
-  final VMIsolate _appIsolate;
+  /// The main isolate hosting the Flutter application.
+  ///
+  /// You can use this to invoke your custom extensions.
```

Comment:
bonus points: an example of how to do that would probably be great here.

Reply:
I gave links to the APIs. I think an example is probably out of scope for this PR since this is not a simple use case.